desktops/kde.txt

```diff
@@ -0,0 +1,437 @@
+    problems. If you would like to skip building them and instead install the
+    prebuilt packages provided on the GitHub page, install the dependencies 
+    and do
+
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   $ ver=2020.08                                                          |
+    |   $ qtver=5.15.0-1                                                       | 
+    |   $ url=https://github.com/dilyn-corner/KISS-kde/releases/download/$ver  |
+    |                                                                          |
+    |   $ wget $url/qt5.$qtver.tar.gz \                                        |
+    |          -O qt5\#$qtver.tar.gz                                           |
+    |   $ wget $url/qt5-declarative.$qtver.tar.gz \                            |
+    |         -O qt5-declarative\#$qtver.tar.gz                                |
+    |   $ wget $url/qt5-webengine.$qtver.tar.gz \                              |
+    |          -O qt5-webengine\#$qtver.tar.gz                                 |
+    |                                                                          |
+    |   $ kiss i qt5                                                           |
+    |   $ kiss i qt5-declarative                                               |
+    |   $ kiss i qt5-webengine                                                 |
+    |                                                                          |
+    +--------------------------------------------------------------------------+
+
+
+    [2.2] Installing
+    ____________________________________________________________________________
+
+    Now that all of the build requirements are taken care of, we can install 
+    the desktop environment. Make your choice between plasma-desktop or plasma, 
+    and build them by
+
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   $ kiss b plasma-desktop                                                | 
+    |   $ kiss i plasma-desktop                                                | 
+    |                                                                          |
+    +--------------------------------------------------------------------------+
+
+    Take care of any circular dependencies:
+
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   $ kiss b elogind                                                       |
+    |   $ kiss i elogind                                                       |
+    |                                                                          |
+    +--------------------------------------------------------------------------+
```

Comment:
I would like to see where between which packages this cirular dependency is.

Reply:
I believe I saw a circularity between `elogind` and `pam`; I will double check that.

Reply:
Upon further examination, it does not seem that this is the case. I will drop this. Thank you!